Routes/index.js

```diff
@@ -0,0 +1,28 @@
+//Seccion 1. importaciones
+
+const express  =require("express")
+const res = require("express/lib/response")
+const router = express.Router()
+
+//Seccion 2. Rute
+
+router.get("/", (req,res)=> {
+
+    res.render("Hola de nuevo")
```

Comment:
Creo que aca querias enviar el mensaje recuerda que render busca en todos tus hbs el que coincida como o existe "Holadenuevo.hbs" lanza el error de hecho no es un nombre valido por los espacios del string 👀 
```suggestion
    res.send("Hola de nuevo")
```